lib/Doctrine/Connection/Statement.php

```diff
@@ -317,7 +317,7 @@ public function execute($params = array())
      */
     public function fetch($fetchMode = Doctrine_Core::FETCH_BOTH,
                           $cursorOrientation = Doctrine_Core::FETCH_ORI_NEXT,
-                          $cursorOffset = null)
+                          $cursorOffset = 0)
```

Comment:
As it is, `null` on the interface, this is a violation.

Can you try to fix without this violation?